waspc/data/Generator/templates/react-app/src/api.js

```diff
@@ -25,10 +29,16 @@ export const clearAuthToken = () => {
   window.localStorage && window.localStorage.removeItem(WASP_APP_AUTH_TOKEN_NAME)
 }
 
-export const clearLocalStorage = () => {
+export const cleanupStorage = () => {
```

Comment:
So ok, this is now not any local storage, this is instead storage for the API thingy right? Ok hm! Maybe we can name it even more descriptive in that case, smth like cleanupApiCache ? Surprising that we had such bad abstraction before hm, I think I did that, pretty bad!

Reply:
> Surprising that we had such bad abstraction before hm, I think I did that, pretty bad

<img src="https://user-images.githubusercontent.com/22752307/225914685-056f8180-f79b-440c-9396-b043282f1efc.png" alt="image" width=200>